pkg/catalog/catalog.go

```diff
@@ -2741,6 +2740,9 @@ func (c *Catalog) CopyEntry(ctx context.Context, srcRepository, srcRef, srcPath,
 		return nil, err
 	}
 
+	// Update creation date only after actual copy!!!
```

Comment:
```suggestion
	// Update creation date only after actual copy to prevent potential discrepancy between lakefs and the underlying blockstore mtimes
```

Reply:
This comment is incorrect, it does not prevent any discrepancy

Reply:
Huh, this is kind of what I understood from reading the issue description. Maybe "prevent" isn't accurate, as this fix aims to minimise the time difference rather than making it equal, but maybe I got it all wrong 😄
In any case, my point is to add a _why_ for this comment, not only _what_.